tutorials/advanced-sql-migration-tool-ora-hana/advanced-sql-migration-tool-ora-hana.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Convert Oracle-to-SAP HANA with Advanced SQL Migration Tool
+description: Convert an Oracle schema used by a custom application to SAP HANA using SAP Advanced SQL Migration tool
+auto_validation: true
+time: 40
+tags: [ tutorial>beginner, programming-tool>sql, products>sap-Hana]
+primary_tag: programming-tool>sql
+---
+
+## Prerequisites
+ - Oracle DB instance (version >= 11), where a new user and SQL objects will be created with a script provided in the tutorial
+ - Oracle JDBC driver (ojdbc7.jar)
+ - SAP HANA or SAP HANA Cloud instance. Minimum versions are 2.0 rev31 and QRC3-2021 respectively.
+ - SAP Advanced SQL Migration tool installed and configured properly as indicated in the installation guide:  <https://dam.sap.com/mac/app/p/pdf/asset/preview/cLmo5vJ?ltr=a>
+ - Java >= 1.8 installed in the host where the migration tool is executed
+
+## Details
+### You will learn
+- Convert Oracle schema to SAP HANA
+
+The SAP Advanced SQL Migration tool is SAP's migration tool to migrate custom applications running on top of non-SAP databases into SAP databases.
+
+![Select Data Source](ad-sql-mig.png)
```

Comment:
Add border

Reply:
Done

Reply:
There is still no border.
It should be !![Select Data Source](ad-sql-mig.png)

Reply:
I see ... what I did was to add the border in the picture itself. I have removed the border from the picture and add that second "!" you are pointing